seed/static/seed/js/controllers/inventory_map_controller.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * :copyright (c) 2014 - 2018, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.
+ * :author
+ */
+angular.module('BE.seed.controller.inventory_map', [])
+  .controller('inventory_map_controller', [
+    '$scope',
+    '$stateParams',
+    'inventory',
+    'urls',
+    'spinner_utility',
+    function ($scope,
+              $stateParams,
+              inventory,
+              urls,
+              spinner_utility) {
+      spinner_utility.show();
+
+      $scope.inventory_type = $stateParams.inventory_type;
+      $scope.data = inventory.results;
+      $scope.pagination = inventory.pagination;
+
+      $scope.geocoded_data = $scope.data.filter(building => building.long_lat);
+      $scope.ungeocoded_data = $scope.data.filter(building => !building.long_lat);
+
+      // Render Map
+      var renderMap = function () {
+        var raster = new ol.layer.Tile({
+          source: new ol.source.OSM()
+        });
+
+        var vector_style = new ol.style.Style({
+          image: new ol.style.Icon({
+            src: urls.static_url + "seed/images/favicon.ico",
+            scale: 0.3,
+            anchor: [0.5, 1]
+          })
+        });
+
+        var vectors = new ol.layer.Vector({
+          source: vectorSources(),
+          style: vector_style
+        });
+
+        // Consider making center be the centroid of shown points
+        // and make zoom dynamic (just large enough to see all points)
+        center_zoom = {
+          center: ol.proj.fromLonLat([-104.986292, 39.765566]),
```

Comment:
@adrian-lara Is this a hard code of the center?

Reply:
For now, yes. PostGIS has the ability to perform different calculations, and I believe that finding a centroid is possible. I'm less sure about the zoom level, but I left the comment above this `center_zoom` variable (which I'm now realizing I forgot to locally scope, so I'll be changing that too).

For now, this is hard coded to the center of the US, but I'll update the location and zoom to be more dynamic.

Reply:
@adrian-lara cool, maybe make the comment a `TODO: ` Just so we know we need to do something with that later :)